### PR TITLE
[1LP][RFR] tenant mapping

### PR DESCRIPTION
**Pull request by rrasouli**
Since 5.7 tenant mapping button has been added.
True, False
Default value: False
tenant_mapping is the key to be in cfme_data.yaml file

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit cd69a07012445b0d018a5f6296960e4aded9c4c8:

`cfme/cloud/provider/openstack.py`:
- :bangbang: Line 52:27: **E241** *multiple spaces after ':'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3953b0c0cf865e7723831de552be1bdffe3c8c12:

`cfme/cloud/provider/openstack.py`:
- :bangbang: Line 52:27: **E241** *multiple spaces after ':'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by rrasouli** [comment]
@psav @dajoRH please review and merge

**Comment by dajoRH** [comment]
Lint report for commit 88819c8888c783a7d23b0638add2612ad2acc98c:

`cfme/cloud/provider/__init__.py`:
- :bangbang: Line 79:65: **E202** *whitespace before ')'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/cloud/provider/__init__.py [hunk 1]
```diff
@@ -68,6 +68,7 @@
     fields=[
         ('type_select', AngularSelect("ems_type")),
         ('name_text', Input("name")),
+        ('ten_mapping', CFMECheckbox("ems_tenant_mapping_enabled"), {"appeared_in": "5.7"}),
```
Comment (psav) [inline cfme/cloud/provider/__init__.py]: ```'tenant_mapping``` please

**Pull request merged**